requirements.txt

```diff
@@ -1,4 +1,6 @@
 discord.py
 redis
 javascript
-requests
\ No newline at end of file
+requests
+pandas
+matplotlib
```

Comment:
Any chance we could make that graph optional?

Reply:
Why would we want to do that?
My testing shows that the new requirements only increase ram usage by 5%. (Total is 43 mb)

Reply:
Further testing shows that it is not really needed